nengo_loihi/tests/test_simulator.py

```diff
@@ -46,7 +47,11 @@ def test_dt(dt, a_on_chip, Simulator, seed, plt, allclose):
         nengo.Connection(a, b, function=function,
                          solver=nengo.solvers.LstsqL2(weights=True))
 
-    with Simulator(model, dt=dt, precompute=False) as sim:
+    loihi_model = nengo_loihi.builder.Model(dt=dt)
+    if dt > 1. / (loihi_model.inter_rate * loihi_model.inter_n):
+        loihi_model.inter_rate = 1. / (dt * loihi_model.inter_n)
```

Comment:
Should this be something we're doing in all models (adjusting inter_rate for larger dts)?  That's not something that users are likely to know how to do, so if it's required to use dt>0.001 we should do it automatically, or if that isn't possible then raise a warning telling users how to adjust inter_rate.

Reply:
In that case, I would just set `inter_rate` to be `1. / (dt * loihi_model.inter_n)` by default. Maybe a sensible thing to do.